code/modules/mob/living/carbon/superheroes.dm

```diff
@@ -0,0 +1,181 @@
+/datum/game_mode
+	var/list/datum/mind/superheroes = list()
+	var/list/datum/mind/supervillains = list()
+	var/list/datum/mind/greyshirts = list()
+
+/datum/superheroes
+	var/name
+	var/class
+	var/list/default_genes = list()
+	var/list/default_spells = list()
+
+/datum/superheroes/proc/create(var/mob/living/carbon/human/H)
+	assign_genes(H)
+	assign_spells(H)
+	equip(H)
+	assign_id(H)
+
+/datum/superheroes/proc/equip(var/mob/living/carbon/human/H)
+	H.fully_replace_character_name(H.real_name, name)
+	for(var/obj/item/W in H)
+		if(istype(W,/obj/item/organ)) continue
+		H.unEquip(W)
+
+/datum/superheroes/proc/assign_genes(var/mob/living/carbon/human/H)
+	if(default_genes.len)
+		for(var/gene in default_genes)
+			H.mutations |= gene
+		H.update_mutations()
+
+/datum/superheroes/proc/assign_spells(var/mob/living/carbon/human/H)
+	if(default_spells.len)
+		for(var/spell in default_spells)
+			var/obj/effect/proc_holder/spell/wizard/S = spell
+			if(!S) return
+			H.spell_list += new S
+			H.update_power_buttons()
+
+/datum/superheroes/proc/assign_id(var/mob/living/carbon/human/H)
+	var/obj/item/weapon/card/id/syndicate/W = new(H)
+	W.registered_name = H.real_name
+	W.access = get_all_accesses()
+	if(class == "Superhero")
+		W.name = "[H.real_name]'s ID Card (Superhero)"
+		W.assignment = "Superhero"
+		ticker.mode.superheroes += H.mind
+	else if(class == "Supervillain")
+		W.name = "[H.real_name]'s ID Card (Supervillain)"
+		W.assignment = "Supervillain"
+		ticker.mode.supervillains += H.mind
+
+	H.equip_to_slot_or_del(W, slot_wear_id)
+	H.regenerate_icons()
+
+/datum/superheroes/owlman
+	name = "Owlman"
+	default_genes = list(REGEN, NO_BREATH)
```

Comment:
Might want to give him cold resistance, since wishgranters will often be found on the mining asteroid.